etl/steps/data/grapher/climate/2024-11-19/total_precipitation_annual.meta.yml

```diff
@@ -0,0 +1,44 @@
+    processing_level: major
+  common_processing: |-
+    - Initially, the dataset is provided with specific coordinates in terms of longitude and latitude. To tailor this data to each country, we use geographical boundaries as defined by the World Bank. The method involves trimming the precipitation dataset to match the exact geographical shape of each country. To correct for potential distortions caused by projecting the Earth's curved surface onto a flat map, we apply a latitude-based weighting. This step is essential for maintaining accuracy, particularly in high-latitude regions where distortion is more pronounced. The result of this process is a latitude-weighted average precipitation for each nation.
+    - It’s important to note, however, that due to the resolution constraints of the Copernicus dataset, this methodology might not be as effective for countries with very small landmasses. In such cases, the process may not yield reliable data.
+    - The derived precipitation for each country is calculated based on administrative borders, encompassing all land surface types within these areas. As a result, precipitation over oceans and seas is not included in these averages, keeping the data focused on terrestrial environments.
+    - Global precipitation averages and anomalies, however, are calculated over both land and ocean surfaces.
+  precipitation_anomaly: |-
+    - The precipitation anomaly is calculated by comparing the average precipitation of a specific time period (e.g., a particular year or month) to the average surface precipitation of the same period from 1991 to 2020.
+    - When calculating anomalies for each country, the total precipitation of a given year or month is compared to the 1991-2020 average precipitation for that specific country.
+    - The reason for using the 1991-2020 period as the reference mean is that it is the standard reference period used by our data source, the Copernicus Climate Change Service. This period is also adopted by the UK Met Office. This approach ensures consistency in identifying climate variations over time.
+
+
+dataset:
+  title: Annual precipitation and anomalies by country
+  update_period_days: 31
```

Comment:
Sorry, why is this number different from garden?

Reply:
just forgot to change this one! sorry done now!